### PR TITLE
[clang-tidy] add explicit to single argument constructors

### PR DESCRIPTION
**Pull request by neheb**
Found with google-explicit-constructor

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by jonesmz** [comment]
:+1:

File: src/build_test.cc [hunk 1]
```diff
@@ -488,7 +488,7 @@ struct BuildTest : public StateTestWithBuiltinRules, public BuildLogUser {
                 status_(config_) {
   }
 
-  BuildTest(DepsLog* log) : config_(MakeConfig()), command_runner_(&fs_),
+  explicit BuildTest(DepsLog* log) : config_(MakeConfig()), command_runner_(&fs_),
                             builder_(&state_, config_, NULL, log, &fs_),
```
Comment (jhasse) [inline src/build_test.cc:492]: clang-format

**Review by jhasse — CHANGES_REQUESTED** [review]
Can you add it to .clang-tidy, too?

**Comment by neheb** [comment]
Done.

**Comment by neheb** [comment]
I think I'll remove that.

It also complains on string_piece.h , which causes all sorts of errors.

**Pull request merged**